valgrind/launch-test-instance

```diff
@@ -41,7 +53,13 @@ valgrind_instance_ip=$(aws ec2 describe-instances \
 # run valgrind tests
 echo "Running the valgrind tests..."
 echo "This will take hours, test results will be sent via e-mail."
-ssh -o StrictHostKeyChecking=no -i $key_name.pem ubuntu@$valgrind_instance_ip "screen -d -m run-valgrind-tests"
+ssh -o StrictHostKeyChecking=no -i $key_name.pem ubuntu@$valgrind_instance_ip 'REPORT_EMAIL="${REPORT_EMAIL}" POSTGRES_GITREF="${POSTGRES_GITREF}" CITUS_GITREF="${CITUS_GITREF}" srceen -d -m run-valgrind-tests'
+
+echo login:
+echo "  " ssh -o StrictHostKeyChecking=no -i $key_name.pem ubuntu@$valgrind_instance_ip
+echo
 
 #delete the key pair after we are done with tests
-aws ec2 delete-key-pair --key-name $key_name
+echo "please run when done"
```

Comment:
No need for this. Even if we delete the key pair automatically, the instance created with the specific key pair can be accessible with that key pair.

Reply:
tru, this was for me during development, will revert back